cobbler.spec

```diff
@@ -307,6 +316,11 @@ rm %{buildroot}%{_sysconfdir}/cobbler/cobbler.conf
 mkdir -p %{buildroot}%{_sysconfdir}/logrotate.d
 mv %{buildroot}%{_sysconfdir}/cobbler/cobblerd_rotate %{buildroot}%{_sysconfdir}/logrotate.d/cobblerd
 
+# No VirtualHost definition as it overwrite all the Uyuni RewriteRules configured in conf.d/
+mkdir -p %{buildroot}%{_sysconfdir}/apache2/conf.d
+mv %{buildroot}%{_sysconfdir}/apache2/vhosts.d/cobbler.conf %{buildroot}%{_sysconfdir}/apache2/conf.d
+sed -i 's/^.*VirtualHost.*$//g' %{buildroot}%{_sysconfdir}/apache2/conf.d/cobbler.conf
```

Comment:
This is not acceptable to do. What is the actual problem here, that Uyuni broke compatibility with Cobbler-Web? I don't think Cobbler-Web has ever been supported with Uyuni (or Spacewalk before that).

Reply:
This is not about `cobbler-web`. It's the proxy config for cobblerd.

Reply:
Okay, let's drop this, since this is Uyuni/SUMA specific.